parser/parser.c

```diff
@@ -57,6 +57,12 @@ PARSER *parser_init(RRDHOST *host, void *user, void *input, PARSER_INPUT_TYPE fl
         parser_add_keyword(parser, "CLABEL", pluginsd_clabel);
         parser_add_keyword(parser, PLUGINSD_KEYWORD_BEGIN, pluginsd_begin);
         parser_add_keyword(parser, "SET", pluginsd_set);
+        
+        parser_add_keyword(parser, PLUGINSD_KEYWORD_REP, pluginsd_rep);
+        parser_add_keyword(parser, PLUGINSD_KEYWORD_GAP, pluginsd_gap);
+        parser_add_keyword(parser, PLUGINSD_KEYWORD_RDATA, pluginsd_rdata);
+        parser_add_keyword(parser, PLUGINSD_KEYWORD_FILL, pluginsd_fill);
+        parser_add_keyword(parser, PLUGINSD_KEYWORD_FILLEND, pluginsd_fill_end);
```

Comment:
I am observing here that you are using 5 of the keywords that we defined inside `collectors/plugins.d/plugins_d.h`, the following are missed:

```c
#define PLUGINSD_KEYWORD_REP_ON "ON"
#define PLUGINSD_KEYWORD_REP_OFF "OFF"
#define PLUGINSD_KEYWORD_REP_ACK "ACK"
```

on the other hand I see that we have an `enum` with these values:

```c
typedef enum rep_arg {
    REP_ARG_ERROR = 0,
    REP_OFF = 1,
    REP_ON = 2,
    REP_ACK = 3
} REP_ARG;
```

And we treat the "missing" keywords inside file  `collectors/plugins.d/pluginsd_parser.c` . 

I understood that in theory all the `8` are keywords, because I am reviewing this PR, but when it is merged and in the future we try to review another PR that can complement this, we can be lost. I suggest to rename the three keywords that are not used here for another name, because to use the same prefix can be a problem.